lib/main.coffee

```diff
@@ -32,13 +41,44 @@ module.exports =
     @subs.add atom.config.observe 'linter-sass-lint.configFile',
       (configFile) =>
         @configFile = configFile
-    @subs.add atom.config.observe 'linter-sass-lint.executablePath',
-      (executablePath) =>
-        @executablePath = executablePath
+    @subs.add atom.config.observe 'linter-sass-lint.globalSassLint',
+      (globalSassLint) =>
+        @globalSassLint = globalSassLint
+    @subs.add atom.config.observe 'linter-sass-lint.globalNodePath',
+      (globalNodePath) =>
+        @globalPath = globalNodePath
 
   deactivate: ->
     @subs.dispose()
 
+  # return a relative path for a file within our project
+  # we use this to match it to our include/exclude glob string within sass-lint's
+  # user specified config
+  getFilePath: (path) ->
+    relative = atom.project.relativizePath(path)
+
+  # Determines whether to use the sass-lint package included with linter-sass-lint
+  # or the users globally installed sass-lint version
+  findExecutable: ->
+    if not @globalSassLint
+      return require path.join(__dirname, '..', 'node_modules', 'sass-lint')
+    if @globalPath is '' and prefixPath is null
+      npmCommand = if process.platform is 'win32' then 'npm.cmd' else 'npm'
+      try
+        # Atom on OSX doesn't inherit a users PATH unless opened from the command line
+        if process.platform isnt 'win32'
+        then env = Object.assign({}, process.env, {PATH: getPath()})
```

Comment:
Just a note that if the OS isn't OSX, `consistent-path::getPath` doesn't modify the global path and returns it as-is